src/twisted/newsfragments/11885.misc

```diff
@@ -0,0 +1 @@
+Fix circular references when using `defer.inlineCallbacks`.
```

Comment:
Maybe we can call it a feature for CPython ... and name this file 11885.feature

```suggestion
`twisted.internet.defer.inlineCallbacks`, when using CPython`, after the deferred is called, it no longer keeps a reference to the wrapped function and arguments.
```

Reply:
I've changed it to be a feature. I've tried to word it so that library users can better understand the impact, but I'm not sure if it's just too wordy. 

I'm happy for you to just change it if you want to. Words aren't my forte.